api/types/types.go

```diff
@@ -304,9 +304,10 @@ type HealthcheckResult struct {
 
 // Health states
 const (
-	Starting  = "starting"  // Starting indicates that the container is not yet ready
-	Healthy   = "healthy"   // Healthy indicates that the container is running correctly
-	Unhealthy = "unhealthy" // Unhealthy indicates that the container has a problem
+	NoHealthcheck = "no healthcheck" // Indicates there is no healthcheck
```

Comment:
just looking at this; having a space in this value could make it less easy to use (now I _must_ quote the filter), should we use (e.g.) "none", or "no-healthcheck" ?